pandas/io/pytables.py

```diff
@@ -4649,12 +4661,15 @@ def _set_tz(values, tz, preserve_UTC: bool = False, coerce: bool = False):
     return values
 
 
-def _convert_index(index, encoding=None, errors="strict", format_type=None):
+def _convert_index(name: str, index, encoding=None, errors="strict", format_type=None):
```

Comment:
encoding and errors are I believe str if wanted to add here; I assume `index` might be an instance of `Index` though if non-trivial to verify certainly leave out

Reply:
ive got another branch going that i think gets these.  tried to keep this one pretty narrow since its not _just_ annotations